### PR TITLE
add support for python 36, and install for freeBSD

### PR DESCRIPTION
**Pull request by voileux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ yum install python-argparse python-enum34 python34-requests
 yum install python-enum34 python-requests
 ```
 
+**Installation on FreeBSD**
+```
+yum install py36-requests
+```
```
Comment (nbuchwitz) [inline README.md]: Yum looks wrong for FreeBSD. Shouldn't it be pkg?

**Comment by voileux** [comment]
Yes, pkg is better for freeBSD,

**Comment by voileux** [comment]
I had new commit to correct it

**Comment by voileux** [comment]
Thank's too

**Pull request merged**